3D-microservices-observability/node-js-app/beachshirt/controllers/deliveryController.js

```diff
@@ -0,0 +1,52 @@
+/**
+* Driver for delivery service which is responsible to deliver the shirts of a given style.
+* @author Yogesh Prasad Kurmi (ykurmi@vmware.com)
+*/
+const uuidv1 = require('uuid/v1');
+
+module.exports = function(app, log){
+
+    app.post('/dispatch/:orderNum/', function(req, res){
+        orderNum = req.params.orderNum;
+        shirts = req.body.shirts;
+        if (Math.floor(Math.random() * 5) == 0) {
+            msg = "Failed to order shirts!";
+            log.error(msg);
+            return res.status(503).json({ error: msg });
+        }
+
+        if (Math.floor(Math.random() * 10) == 0) {
+            orderNum = null;
+        }
```

Comment:
I really don't understand the purpose of L12 and L18.  The purpose is to randomly return and error or set the orderNum to null, then add a comment so the reader or developer knows the intention.